Storage/api.h

```diff
@@ -219,6 +219,13 @@ STORAGE_HANDLERS_MAP_ENTRY GenerateRESTfulHandler(STORAGE& storage, const std::s
 }
 
 }  // namespace impl
+template <bool CALL>
```

Comment:
Right now, "ConditionalOn<false>::Call(...) executes whatever is passed to it. It really should be ConditionalOn<true>::Call(...) that executes, and <false> that should elide the lambda.